packages/@atjson/editor/src/components/editable-link.ts

```diff
@@ -0,0 +1,62 @@
+import EditableComponent from '../mixins/editable-component';
+import LinkEditor from './link-editor';
+
+if (!window.customElements.get('link-editor')) {
+  window.customElements.define('link-editor', LinkEditor);
+}
+
+export default class EditableLink extends EditableComponent {
+
+  static template = `<div class="controls"><link-editor></link-editor></div><a class="text-link" target="_blank"><slot></slot></a>`;
+
+  static observedAttributes = ['url', 'nofollow'];
+
+  static style = EditableComponent.style + `
+    a {
+      text-decoration: underline;
+      color: blue;
+      cursor: text;
+    }
+  `;
+
+  static events = Object.assign({
+    'click .text-link': 'cancelLinkClick',
+  }, EditableComponent.events);
+
+  static annotationName = 'link';
+
+  static elementRenderer = (node) => {
+    let link = document.createElement('editable-link');
```

Comment:
Yup, 100%

Reply:
I didn't build that out because there's some question of how attributes get passed down to the web component and back up through the events, and figure Do Repeat Yourself applies here. :D